modules/microshift-install-rhel-image-mode-build-image.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-build-image_{context}"]
+= Building the bootc image
+
+Build your Red Hat Enterprise Linux as a bootable container image by using a `ContainerFile` and include {microshift-short}.
+
+[NOTE]
+====
+Image mode for {op-system-base-full} is technology preview. Using in production environments is not supported.
+====
+
+.Prerequisites
+* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
+* You are logged into the {op-system} 9.4 host using the user credentials that have SUDO permissions configured.
+* The `rhocp` and `fast-datapath` repositories are enabled so that {microshift-short} and its dependencies can be installed. These repositories must be accessible in the host subscription, but do not necessarily need to be enabled on the host.
+* You have a remote registry such as link:https:/quay.io[Red Hat quay] for storing and accessing bootc images.
+//Q: do we need this one since we aren't documenting running the VM?
+* A physical hypervisor host with the `libvirt` virtualization platform for starting virtual machines that run {op-system} operating system containing {microshift-short} binaries. See link:https://libvirt.org/[libvirt Virtualization API] for more information.
+
+.Procedure
+
+//Do we want to give a command for this?
+//Does it matter where the file is or what the name is?
+. Create a `ContainerFile` and include the following contents:
```

Comment:
File name matters because it's used subsequently in commands.
```suggestion
. Create a `Containerfile` file and include the following contents:
```